src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar I was kidding you :wink: 
The fact seem that it is now in all the Zeranoe, FFmpeg build by default, so i suspect that perhaps you need to ensure Bios or UEFI settings force it to present both to the OS ?

**EDIT**: Oups..my bad :flushed: 
ffmpeg64.exe -y -hwaccel auto -c:v hevc_cuvid -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6M -maxrate 6M -b:v 3M -c:v h264_nvenc -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f ismv -movflags isml+frag_keyframe hevc_test1.mp4 -benchmark